public/app/features/expressions/components/Resample.tsx

```diff
@@ -8,11 +8,11 @@ import { downsamplingTypes, ExpressionQuery, upsamplingTypes } from '../types';
 interface Props {
   refIds: Array<SelectableValue<string>>;
   query: ExpressionQuery;
-  labelWidth: number;
+  labelWidth?: number | 'auto';
```

Comment:
💬 The downstream component supports `auto` as `labelWidth` but this not supported in the typing of this component.